react/src/components/ModelTryContentButton.tsx

```diff
@@ -0,0 +1,492 @@
+          resources: {
+            // FIXME: manually convert to string since server-side only allows [str,str] tuple
+            cpu: values.resource.cpu.toString(),
+            mem: values.resource.mem,
+            ...(values.resource.accelerator > 0
+              ? {
+                  [values.resource.acceleratorType]:
+                    // FIXME: manually convert to string since server-side only allows [str,str] tuple
+                    values.resource.accelerator.toString(),
+                }
+              : undefined),
+          },
+          resource_opts: {
+            shmem:
+              compareNumberWithUnits(values.resource.mem, '4g') > 0 &&
+              compareNumberWithUnits(values.resource.shmem, '1g') < 0
+                ? '1g'
+                : values.resource.shmem,
+          },
+        },
+      };
+      return baiSignedRequestWithPromise({
+        method: 'POST',
+        url: '/services',
+        body,
+        client: baiClient,
+      });
+    },
+  });
+
+  const getServiceInputByModelNameAndVFolderId = (
+    modelName: string,
+    vfolderID: string,
+  ): ServiceLauncherFormValue => {
+    return {
+      serviceName: `${modelName}-${generateRandomString(4)}`,
+      replicas: 1,
+      environments: modelCardMetadata?.environments || {
+        environment: '',
+        version: '',
+        image: null,
+      },
+      runtimeVariant: modelCardMetadata?.runtimeVariant || 'custom',
+      cluster_size: 1,
+      cluster_mode: 'single-node',
+      openToPublic: true,
+      resourceGroup: currentResourceGroupByProject as string,
+      resource: modelCardMetadata?.resource || {
+        cpu: 4,
+        mem: '32g',
+        accelerator: 10,
+        acceleratorType: 'cuda.shares',
+        shmem: '1g',
+      },
+      vFolderID: vfolderID, // TODO: add cloned folder result
+      modelMountDestination: '/models',
+      modelDefinitionPath: '',
+      vfoldersAliasMap: {},
+      envvars: modelCardMetadata?.envvars || [],
+      enabledAutomaticShmem: false,
+    };
+  };
+
+  const cloneOrCreateModelService = (runtimeVariant: string) => {
+    let modelId = 'vllm-model';
+    switch (runtimeVariant) {
+      case 'vllm':
+      default:
+        break;
+      case 'nim':
+        modelId = 'nim-model';
+        break;
+      case 'custom':
+        modelId = 'custom';
+        break;
+    }
+    if (filteredModelStoreList.length < 1) {
+      mutationToClone.mutate(
+        {
+          input: {
+            permission: 'wd', // write-delete permission
+            target_host: modelStorageHost, // lowestUsageHost, // clone to accessible and lowest usage storage host
+            target_name: `${modelCardMetadata?.serviceName || modelName}`, // TODO: add suffix to avoid name conflict
+            usage_mode: 'model',
+          },
+          name: `${modelName === 'Talkativot UI' ? 'talkativot-standalone' : modelCardMetadata?.serviceName}`,
```

Comment:
@agatha197 Probably it's originated from this code, but I think it's better to remove both of them, for code quality. We'll handle this special models into model_card_metadata, anyway.